### DISCUSSION TITLE
Mesh Type

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12577] [General]
Dear contributors,

I've been getting started with this software recently, my question is which types of mesh does FDS apply(say, tetrahedral, Wedge-shaped body, and polyhedral) and how does FDS work or what is the flow chart of this procedure?

Kindly regards,
Li

**Comment by drjfloyd** [comment]
Converting to a dicsussion

**Comment by drjfloyd** [comment]
Please take the time to read the FDS documentation inlcuding the Technical Reference Manual, User's Guide, and the Verificaiton and Validaiton Guides.
